cmd/plume/release.go

```diff
@@ -518,25 +518,62 @@ func modifyReleaseMetadataIndex(spec *fcosChannelSpec, commitId string) {
 		plog.Fatalf("unmarshaling release metadata json: %v", err)
 	}
 
-	url, err := url.Parse(fmt.Sprintf("https://%s.s3.amazonaws.com/prod/streams/%s/builds/%s/release.json", spec.Bucket, specChannel, specVersion))
+	releasePath := filepath.Join("prod", "streams", specChannel, "builds", specVersion, "release.json")
+	url, err := url.Parse(fmt.Sprintf("https://%s.s3.amazonaws.com/%s", spec.Bucket, releasePath))
 	if err != nil {
 		plog.Fatalf("creating metadata url: %v", err)
 	}
 
+	releaseFile, err := api.DownloadFile(spec.Bucket, releasePath)
+	if err != nil {
+		plog.Fatalf("downloading release metadata: %v", err)
+	}
+	defer releaseFile.Close()
+
+	releaseData, err := ioutil.ReadAll(releaseFile)
+	if err != nil {
+		plog.Fatalf("reading release metadata: %v", err)
+	}
+
+	var im IndividualReleaseMetadata
+	err = json.Unmarshal(releaseData, &im)
+	if err != nil {
+		plog.Fatalf("unmarshaling release metadata: %v", err)
+	}
+
+	var commits []Commit
+	for arch, vals := range im.Architectures {
+		commits = append(commits, Commit{
+			Architecture: arch,
+			Checksum:     vals.Commit,
```

Comment:
@jlebon just double-checking: is it ok if we call this field (the ostree commit SHA) `commit.checksum`, or is there some possible confusion with `ostree-content-checksum` (which I'm not sure what it is)?

Reply:
Yup, checksum is fine. In any context where we're talking about the content checksum, we'd explicitly preface it with "content". For more information of what it is, see https://github.com/ostreedev/ostree/pull/1449.